packages/expo-screen-capture/android/src/main/java/expo/modules/screencapture/ScreenCaptureModule.kt

```diff
@@ -30,6 +30,7 @@ class ScreenCaptureModule : Module() {
     get() = safeCurrentActivity ?: throw Exceptions.MissingActivity()
   private var screenCaptureCallback: Activity.ScreenCaptureCallback? = null
   private var screenshotEventEmitter: ScreenshotEventEmitter? = null
+  private var screenRecordingEventEmitter: ScreenRecordingEventEmitter? = null
```

Comment:
I think it may be possible to implement this for Android 15+
https://developer.android.com/about/versions/15/features#screen-recording-detection

Reply:
Same comment about splitting this work into a separate PR